### ISSUE TITLE
Hl7.Fhir.R4B v3.2.0 NuGet package doesn't contain SearchParamDefinitions

### ISSUE THREAD

**Issue by andy-a-o** [FirelyTeam/firely-net-sdk#1746]
:information_source: Do you want to ask something or start a discussion, then go [here](https://github.com/FirelyTeam/firely-net-sdk/discussions)

**Describe the bug**
We are trying to add R4B functionality into the Spark repo. And we got tests failing because there's no search param defs included into the `Hl7.Fhir.R4B` package. The defs should be generated, but they are empty in the source code:

`Template-ModelInfo.cs`

```
    public static List<SearchParamDefinition> SearchParameters = new List<SearchParamDefinition>()
    {
    };
```

**To Reproduce**
Just list the contents of `ModelInfo.SearchParameters`.

**Expected behavior**
`ModelInfo.SearchParameters` list is not empty.

**Version used:**
 - FHIR Version: R4B
 - Version: 3.2.0

**Comment by ewoutkramer** [comment]
This is indeed the case.  HL7 has forgotten to include the search parameters in the R4B release. We have notified them (see https://jira.hl7.org/browse/FHIR-32782), so I expect that this will be repaired in the next update of FHIR R4B.

**Comment by ewoutkramer** [comment]
Let's keep it open so we can double check that they have indeed added them for the next release.

**Comment by marcovisserFurore** [comment]
See also the [release notes](https://github.com/FirelyTeam/firely-net-sdk/releases/tag/v3.2.0-r4B) for 3.2.0 for R4B and other known issues for this release.

**Comment by marcovisserFurore** [comment]
This has been resolved in release 3.8.0 of this library. See [here](https://github.com/FirelyTeam/firely-net-sdk/releases/tag/v3.8.0-r4B) for more information.
